vscode/src/chat/chat-view/ChatController.ts

```diff
@@ -925,14 +925,21 @@ export class ChatController implements vscode.Disposable, vscode.WebviewViewProv
         this.contextFilesQueryCancellation = cancellation
 
         const source = 'chat'
+
+        // Use enum to send source values to metadata, making this data available on all instances.
+        enum atMentionSourceTelemetryMetadataMapping {
+            chat = 1,
+        }
```

Comment:
@valerybugakov, For this one, would you also recommend converting to const? 

i.e.
```suggestion
        // Use numerical mapping to send source values to metadata, making this data available on all instances.
        const atMentionSourceTelemetryMetadataMapping = {
            chat: 1,
        } as const
```

Reply:
Yeah, the main benefit would be that we can link it to possible `source` values: 

```ts
const source = 'chat'

const atMentionSourceTelemetryMetadataMapping: Record<typeof source, number> = {
  chat: 1,
} as const

Reply:
Thank you for confirming and explaining!